doc/TBG_macros.cfg

```diff
@@ -186,7 +186,8 @@ TBG_checkpoints="--checkpoints 1000"
 
 # Restart the simulation from checkpoints created using TBG_checkpoints
 TBG_restart="--restart --restart-step 1000"
```

Comment:
Could you put `--restart-step 1000` as an optional command line flag to the comment below?